pkg/controller/hyperconverged/hyperconverged_controller_test.go

```diff
@@ -1487,7 +1533,7 @@ var _ = Describe("HyperconvergedController", func() {
 				).To(BeNil())
 
 				Expect(foundResource.Status.RelatedObjects).ToNot(BeNil())
-				Expect(len(foundResource.Status.RelatedObjects)).Should(Equal(8))
+				Expect(len(foundResource.Status.RelatedObjects)).Should(Equal(9))
```

Comment:
note this would be conflicted by:
https://github.com/kubevirt/hyperconverged-cluster-operator/pull/583/files#diff-0c02a176925e0dc236986fb333d93dc7R1337
which will be merged soon.

Reply:
Thanks, I will rebase if needed.